otherlibs/runtime_events/runtime_events_consumer.c

```diff
@@ -149,10 +139,7 @@ caml_runtime_events_create_cursor(const char_os* runtime_events_path, int pid,
     FILE_ATTRIBUTE_NORMAL,
     NULL
   );
-
   if (cursor->ring_file_handle == INVALID_HANDLE_VALUE) {
-    caml_stat_free(cursor);
-    caml_stat_free(runtime_events_loc);
     return E_OPEN_FAILURE;
```

Comment:
For consistency, please set `ret` and then `goto fail_create_file` here (putting that label right before the `return ret`).